### PR TITLE
[MODULAR] [READY] Ashwalker Village 

### PR DESCRIPTION
**Pull request by ForrestWick**
## About The Pull Request

As Citadel did a long while ago, and as we had when Skyrat was still Citcode, Ashwalkers have been given an updated version of their nest in modular_skyrat. The actual nest ruin that was previously there is untouched, and if this winds up getting merged, I'll leave it up to whoever controls map stuff as to whether or not the small nest ruin gets left alone. If it does, I'll come back to it to add some of the QOL upgrades the village has. This PR also adds some new clothing options for Ashwalkers, supplied in the temple roundstart.

The village layout is seen here: https://cdn.discordapp.com/attachments/640744619302453316/866897079225155584/unknown.png

Added new clothes, revamped the layout, added some new appliances (the mortar on the table is a reskinned grinder), an ORM so you don't have to rely on bonfire smelting RNG, a mining vendor (might get removed, we'll see), a larger farming area, an expanded temple, a proper storage hut, and a cooking hut. The rest of the space is ripe for building.

## Why It's Good For The Game

General QOL for Ashwalkers, a lot of it presents more options for how to do things. The map is modular and the normal tg ruin is untouched.. Also new clothes for the lizard dudes.

## Changelog
:cl:
add: Added more ashwalker clothes.
expansion: Made an Ashwalker village that is a permanent fixture of lavaland, expanded the previous ashwalker nest ruin as a base for this.
qol: Gives Ashwalkers more tools, barter options, and general gameplay options for their rounds.
config: put the new map in the config file
/:cl:

### CODE REVIEW

**Comment by SpaceVampire** [comment]
It would still look better if you didn't have 14 pairs of shoes that will never end up getting used.
All the racks on the bottom right are way overstocked, I recall right-clicking on em once.

**Comment by ForrestWick** [comment]
> It would still look better if you didn't have 14 pairs of shoes that will never end up getting used.
> All the racks on the bottom right are way overstocked, I recall right-clicking on em once.

Done

File: tgstation.dme [hunk 1]
```diff
@@ -4336,6 +4342,7 @@
 #include "modular_skyrat\modules\modular_items\code\game\objects\items\cross.dm"
 #include "modular_skyrat\modules\modular_items\code\game\objects\items\miscellaneous.dm"
 #include "modular_skyrat\modules\modular_items\code\game\objects\items\mining\equipment\kinetic_crusher.dm"
+#include "modular_skyrat\modules\modular_items\code\game\objects\items\mining\equipment\necklace.dm"
```
Comment (Gandalf2k15) [inline tgstation.dme]:
```suggestion
```

**Comment by ForrestWick** [comment]
According to Infrey, spawners code is busted right now and needs to be fixed.

**Comment by ForrestWick** [comment]
Redoing PR due to master merge issues

**Pull request closed without merging**